config/locales/en.yml

```
@@ -967,6 +967,16 @@ en:
       default: Selection from a list
       none_of_the_above: None of the above
       only_one_option: Selection from a list, one option only.
+  page_route_card:
+    check_your_answers: Check your answers before submitting
+    conditional_answer_value: '"%{answer_value}"'
```

```suggestion
    conditional_answer_value: '%{answer_value}'
```
Hoping this removes the double quotes around the answer that a route is based on?

---

It didn't but I changed it elsewhere to remove them.